src/cmd.js

```diff
@@ -258,6 +288,46 @@ function action(actionToDispatch) {
   });
 }
 
+function schedule(nestedCmd, delayMs, onScheduledActionCreator) {
+  return _schedule(nestedCmd, delayMs, onScheduledActionCreator, false);
+}
+
+function scheduleRepeating(nestedCmd, delayMs, onScheduledActionCreator) {
+  return _schedule(nestedCmd, delayMs, onScheduledActionCreator, true);
+}
+
+function _schedule(nestedCmd, delayMs, onScheduledActionCreator, isRepeating) {
```

Comment:
since this isn't a publicly accessible function I think the _ name is kind of unnecessary. Just call it whateverHelper or something (depending on the final name of the Cmd(s))